remove django-autocomplete-light static files

MRoci opened the pull request:
Hi everyone,
First of all thank you for efforts into this great project!

That said we recently had compatibility issues updating to recent versions of `django-autocomplete-light`, caused by changes made to static files.
In particular `dal` from 3.7.0 has removed the `autocomplete.init.js` script due to a complete rework of the initialization system, so at the moment trying to load it from `AutocompleteFilter` can cause some requests to fail with 404 or worse if the file is still present as a stale static object (it breaks the widget).

Since all these static assets inherited from `django-autocomplete-light` should be already loaded by `ModelSelect2` widget seems reasonable to remove them from filter's media so that the widget alone can pick the correct ones regardless of the version installed.

xrmx reviewed (changes requested):
Uhm it looks like there are also other changes needed, I had to update the template to use django.jQuery, forward-fix may need the very same fix but I've removed it altogether locally

MRoci committed:
remove django-autocomplete-light static files

 as they are already loaded by the ModelSelect2 widget it is not necessary to re-load them from the autocomplete filter.
 Also in recent versions some of these scripts have been removed and this causes requests with a response code of 404
 or worse can cause the AutocompleteFilter to break.
 For example, since version 3.7.0 of dal the `autocomplete.init.js` script has been removed in favor of a new initialization system,
 but if the file is still present as an obsolete asset it conflicts with the new init system preventing the plugin from loading

MRoci committed:
use django's jquery

since dal doesn't initialize jquery anymore

MRoci commented:
I've updated the PR to use django.jQuery in the template and forward-fix.js.
Now it seems to behave correctly

shamanu4 commented:
@MRoci, @xrmx, Thank you for your contribution. 
Your PR was included in the latest release https://pypi.org/project/dal-admin-filters/1.1.0/
